ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -365,47 +366,53 @@ def get_min_stopping_distance(self, ptr_id):
 
         if a > 0.0:
             # currently speeding up
-            time_factor = 0.9
+            time_factor = 0.8
         else:
             time_factor = self.dyn_jmt_time_factor
 
-        # empirical settings to start closer to goal
-        if a < 1.0:
-            decel_rate = 2.0
-        elif a < 2.0:
-            decel_rate = 1.5
-        elif a < 3.0:
-            decel_rate = 1.35
-        elif a < 3.4:
-            decel_rate = 1.2
+        if math.fabs(max_jerk + self.max_desired_jerk) < 0.1:
```

Comment:
I guess this intention is if the difference between `max_jerk` and `max_desired_jerk` is small, but it is `(max_jerk + self.max_desired_jerk)` because `max_jerk` should be a negative argument?  

It may be better to rename `max_jerk` to `max_neg_jerk` in this function to make the sign intention a little more clear, and the condition could be `if math.fabs(-self.max_desired_jerk - max_neg_jerk) < 0.1:` so that it's more clearly calculating the difference.